### DISCUSSION TITLE
Change location of Ente Auth configuration file

### DISCUSSION THREAD

**Discussion by MarioSertorio** [ente-io/ente#2087] [Enhancements]
Currently, the configuration file is stored in the following location:

**%userprofile%/Documents/ente/**

However, I would like to request that this file be moved to a more appropriate location within the user's AppData folder, such as:

**%userprofile%/AppData/Local/ente/**

The reasons for this request are:

1. User Documents folders are intended for user-created documents and files, not application configuration data.
 
2. Placing the configuration file in the AppData folder follows best practices for separating user data from application data.

Thank you for your consideration and for developing a great application.
Best regards.

**Comment by jwinstonsf** [comment]
That file,along with a temp folder is stored on my desktop which is even less appropriate.

**Comment by Eternal9100** [comment]
> Currently, the configuration file is stored in the following location:
> 
> **%userprofile%/Documents/ente/**

There are some apps which go to "C:\Users\User\Documents" as a means of ensuring that the data is backed up because people backup or sync their docs, but I agree with you in as much as Linux orginated app in windows should either use %appdata% because it is the equivalent of ~/.config/ in Linux, but some e.g. Virtualbox just put it in the root of the home folder. Any is preferable to "Documents" I think. I ask myself the question "Would you do that in Linux?"

**Comment by ua741** [comment]
This should be fixed in the recent version of the app. Please feel free to comment if we missed anything.

**Comment by Eternal9100** [comment]
There is a folder _%userprofile%\Documents\enteauthinit_ still present. It is recreated at every app start. Is this intended? For Auth 4.1.6 on Win10.
